api-reference/beta/resources/directory.md

```diff
@@ -0,0 +1,57 @@
+# directory resource type
```

Comment:
This is not the resource type.  It's a deletedItem (which is essentially a directoryObject).  Check the metadata - deletedItems should inherit from directoryObject.